x/slash/keeper.go

```diff
@@ -189,14 +190,17 @@ func (k Keeper) Slash(ctx sdk.Context, reason string, failedValidator staking.Va
 
 		log.Warnf("Slash validator: %s %x, amount: %s, reason: %s, nonce: %d",
 			candidate.ValAccount, mainchain.Hex2Addr(identity), slashAmount, reason, penalty.Nonce)
```

Comment:
add `enableSlash` to this warn log, e.g., `("... nonce: %d, enabled: %t", ..., enableSlash)`